app/views/layouts/application.html.slim

```diff
@@ -2,8 +2,8 @@ doctype 5
 html
   head
     title App Prototype
-    = stylesheet_link_tag    "application", media: "all"
-    = javascript_include_tag "application"
+    = stylesheet_link_tag    "application", media: "all", 'data-turbolinks-track': "reload"
+    = javascript_include_tag "application", 'data-turbolinks-track': "reload"
```

Comment:
I think this is supposed to be `javascript_pack_tag`?

Reply:
Fixed. This PR doesn't work yet (obviously), I wanted to start with the vanilla changes and build up from there. Thanks for spotting this one!